test/api/document.spec.js

```diff
@@ -0,0 +1,216 @@
+/* eslint-disable no-unused-expressions */
+
+const app = require('../../server');
+const request = require('supertest')(app);
+const expect = require('chai').expect;
+const model = require('../../app/models');
+const helper = require('../testHelper');
+
+const adminRoleParams = helper.testRole;
+const regularRoleParams = helper.testRole2;
+const adminUserParams = helper.testUser;
+const regularUserParams = helper.testUser2;
+const publicDocumentParams = helper.testDocument;
+const privateDocumentParams = helper.testDocument2;
+
+describe('DOCUMENT API', () => {
+  let adminRole, regularRole, adminUser, publicToken, publicDocument, privateUser,
+    privateToken, privateDocument;
+
+  before((done) => {
+    model.Role.bulkCreate([adminRoleParams, regularRoleParams], {
+      returning: true })
+      .then((createdRoles) => {
+        adminRole = createdRoles[0];
+        regularRole = createdRoles[1];
+        adminUserParams.RoleId = adminRole.id;
+        regularUserParams.RoleId = regularRole.id;
+
+        request.post('/users')
+          .send(adminUserParams)
+          .end((error, response) => {
+            adminUser = response.body.newUser;
+            publicToken = response.body.token;
+            done();
+          });
+      });
+  });
+
+  after(() => {
+    return model.sequelize.sync({ force: true });
+  });
+
+  it('should correctly create test roles & user', () => {
+    expect(adminRole.title).to.equal(adminRoleParams.title);
+    expect(regularRole.title).to.equal(regularRoleParams.title);
+    expect(adminUser).to.exist;
+    expect(adminUser.email).to.equal(adminUserParams.email);
+  });
+
+  describe('REQUESTS', () => {
+    beforeEach((done) => {
+      model.Document.create(publicDocumentParams)
+        .then((createdPublicDocument) => {
+          publicDocument = createdPublicDocument;
+          done();
+        });
+    });
+
+    afterEach(() => {
+      return model.Document.destroy({ where: {} });
```

Comment:
I think this is fine if you just want to delete data in the `Documents` table. `.sync()` actually drops and recreate the whole database.